### ISSUE TITLE
Can't select text when rendering to a pixel buffer

### ISSUE THREAD

**Issue by timakro** [ultralight-ux/Ultralight#364]
I followed the guide at https://docs.ultralig.ht/docs/integrating-with-games and it seems I implemented mouse events correctly as I can follow links and use the scroll bar by clicking and dragging. But I'm not able to select text by clicking and dragging (I'd expect the text to be highlighted). My goal is to implement copying text from the browser with CTRL+C.

**Comment by adamjs** [comment]
Hey there! Did you remember to call `View::Focus()` after creating your `View`? Failure to call this will usually result in the "focus" states not updating.

Alternatively, if you are using the latest trunk (1.3), all Views are initially focused (see: [`ViewConfig::initial_focus`](https://github.com/ultralight-ux/Ultralight-API/blob/master/Ultralight/View.h#L44))

**Comment by timakro** [comment]
Yep, I called `ulViewFocus` after creating the view. Now I tried to call it regularly in the main loop but that didn't help eaither. I'm using the C API if that matters and I'm on version 1.2.1. Where can I download 1.3?

**Comment by timakro** [comment]
Interesting, I just found out that double/triple clicking works like expected. But click+drag doesn't
